### PR TITLE
eBPF alerts

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
This is the first PR adding alerts for `eBPF.plugin` charts. For this first group of alerts we are addressing alerts where it is not necessary to add additional charts.

##### Component Name
health.
##### Test Plan

1 - Install branch
2 - Access `https://localhost:19999/api/v1/alarms?all` and verify alerts were applied.

### CODE REVIEW

**Comment by vlvkobal** [comment]
The alarms are very controversial. What is the reason for adding them? We definitely haven't got any requests from our users. Have there been any specific inquiries and reasoning from our product team?

File: collectors/ebpf.plugin/README.md [hunk 1]
```diff
@@ -804,6 +804,29 @@ If they are already mounted, you will see an error. You can also configure your
...
+  our users.
+
+### Silent alerts
+
+Certain Alerts are of high interest for specific use cases due to a safety policy or development purposes. To avoid
+alert overload, 'netdata' set them [silent](/guides/monitor/stop-notifications-alarms).
+
+-   task_error: This alert is raised every time a process/thread cannot be created.
+-   mount_call: For security reasons, some companies create policies that does not allow stick drivers to be used,
+    this alert sends notification every time a `mount (2)` or `umount (2)` `syscall` is called.
```
Comment (ilyam8) [inline collectors/ebpf.plugin/README.md:828]: > some companies create policies that do not allow stick drivers to be used

I think this one is very questionable.

Why do we need to check mount/umount calls? I think it is not a verbal agreement between users/administrators, but some technical constraint (inserting a USB stick doesn't result in `mount `call).

Also, creating/removing a docker container triggers many `mount`/`umount` calls.

File: collectors/ebpf.plugin/README.md [hunk 2]
```diff
@@ -804,6 +804,29 @@ If they are already mounted, you will see an error. You can also configure your
...
+- apps_oomkill: When some apps group has an event of [Out Of Memory](#OOM-Killing), `netdata` sends a notification for
+  our users.
+
+### Silent alerts
+
+Certain Alerts are of high interest for specific use cases due to a safety policy or development purposes. To avoid
+alert overload, 'netdata' set them [silent](/guides/monitor/stop-notifications-alarms).
+
+-   task_error: This alert is raised every time a process/thread cannot be created.
+-   mount_call: For security reasons, some companies create policies that does not allow stick drivers to be used,
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:827]:
```suggestion
-   mount_call: For security reasons, some companies create policies that does not allow stick drivers to be used.
```

File: collectors/ebpf.plugin/README.md (hunk 1 above)
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:828]:
```suggestion
    This alert sends a notification every time a `mount (2)` or `umount (2)` `syscall` is called.
```

File: collectors/ebpf.plugin/README.md [hunk 3]
```diff
@@ -804,6 +804,29 @@ If they are already mounted, you will see an error. You can also configure your
 mount these filesystems on startup. More information can be found in
 the [ftrace documentation](https://www.kernel.org/doc/Documentation/trace/ftrace.txt).
 
+## Alerts
+
+Like other plugins, `eBPF.plugin` also have associated alerts for its charts. You can personalize them modifying
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:809]:
```suggestion
Like other plugins, `eBPF.plugin` also has associated alerts for its charts. You can personalize the alerts by modifying
```

File: collectors/ebpf.plugin/README.md [hunk 4]
```diff
@@ -804,6 +804,29 @@ If they are already mounted, you will see an error. You can also configure your
...
+the file `healt.d/ebpf.conf` using [`edit-config`](/docs/monitor/configure-alarms#edit-health-configuration-files).
+
+### Default alerts
+
+The following alerts are enabled by default:
+
+- retransmit_tcp_error: When at least 2% of packages are retransmitted, `netdata` raises alerts to notify possible
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:816]: I don't have us quoting "Netdata" in other areas (of this page and on learn). We can just say it in plain text.

**Review by DShreve2 — APPROVED** [review]

**Comment by thiagoftsm** [comment]
> The alarms are very controversial. What is the reason for adding them? We definitely haven't got any requests from our users. Have there been any specific inquiries and reasoning from our product team?

These alarms and others that will come after the new release were defined in a meeting with CEO and product team.

File: collectors/ebpf.plugin/README.md (hunk 1 above)
Comment (thiagoftsm) [reply collectors/ebpf.plugin/README.md:828]: This is the motive we are setting it as silent. 
There are companies where users cannot extracted data from their environment, so they cannot mount/umount any stick drive their. I know companies that disable all USB when computers arrives in their building, and only after this users receive them.

File: collectors/ebpf.plugin/README.md (hunk 4 above)
Comment (thiagoftsm) [reply collectors/ebpf.plugin/README.md]: All right, I removed quotes and addressed everything you required. Thanks!

File: collectors/ebpf.plugin/README.md (hunk 1 above)
Comment (ilyam8) [reply collectors/ebpf.plugin/README.md]: mount/umount call is not a warning event in general (default stock alarms), it is not even an info event. As I mentioned: you would get a warning active alarm after every `docker run ...`. I suggest removing it. I don't see how setting it to `silent` justifies it.

**Comment by vlvkobal** [comment]
> These alarms and others that will come after the new release were defined in a meeting with CEO and product team.

I don't agree with these alarms and I won't approve them. But my part is minor 😁.

File: collectors/ebpf.plugin/README.md (hunk 1 above)
Comment (thiagoftsm) [reply collectors/ebpf.plugin/README.md]: @ilyam8 I removed completely the mount alarm and I also extend the time to test retransmitted packets.

**Comment by ilyam8** [comment]
@thiagoftsm do we need to keep this PR open? If yes - why? 😄

**Comment by thiagoftsm** [comment]
I will bring another with alerts after the bug fixes, I agree with you that this does not need to be kept open.

**Pull request closed without merging**